cmd/adhoc/cmd/root.go

```diff
@@ -34,7 +62,7 @@ func onPersistentPreRun(cmd *cobra.Command, args []string) error {
 			return log.Configure(parser)
 		},
 	}
-
+	readFile()
```

Comment:
Either make use of the function array before this call or remove that altogether. It is going to get messy otherwise.